nss-day-cohort-44/rare-rest-api-fanciful-snowflake
# created Posts ViewSet with multiple methods, includes serializers

# Description
created Posts ViewSet with multiple methods, includes serializers
 
## Type of change
Please delete options that are not relevant.


# Testing Instructions

Look over code for obvious errors, in particular please check getting current authenticated user (line 21 etc)

1. `git fetch --all`
1. `git checkout ps-post-views`
1. `serve`
